src/garage/torch/distributions/tanh_normal.py

```diff
@@ -0,0 +1,262 @@
+"""A Gaussian distribution with tanh transformation."""
+import torch
+from torch.distributions import Normal
+from torch.distributions.independent import Independent
+
+
+class TanhNormal(torch.distributions.Distribution):
+    r"""A distribution induced by applying a tanh transformation to a Gaussian random variable.
```

Comment:
this is supposed to fit on one line. the solution is not to also exceed the line length :P

Reply:
But there isn't a way to express this in a shorter way